app/views/groups/team_members_list.scala.html

```diff
@@ -57,8 +57,10 @@
         }
 
 
-@errorHref = {
-@if(form.errors.map(error => error.message).contains("error.select-members.empty")) {members} else {search}
+@errorHref = @{
+    if(form.errors.map(error => error.message).contains("error.select-members.empty"))
+        "members"
+    else "search"
```

Comment:
I'll check if this still works, I had issues when the href would be `#"members"` instead of `#members` because it was reading the whole item as a string - might need to update line 78 with the summaryErrors since it's using `.toString`?

`@summaryErrors(form.errors, Some(errorHref.toString))`

Reply:
there was some whitespace which i removed. it was 
"#
members"

Reply:
hmmm alright well it seems to work, but as I've sent in Slack clients errorHref doesn't 😅